bridge/engine/EngineAudioStream.h

```diff
@@ -18,26 +18,26 @@ struct EngineAudioStream
 {
     static constexpr uint32_t MAX_NEIGHBOUR_COUNT = 128;
 
-    EngineAudioStream(const std::string& endpointId,
-        const size_t endpointIdHash,
-        const uint32_t localSsrc,
-        const utils::Optional<uint32_t>& remoteSsrc,
-        transport::RtcTransport& transport,
-        const bool audioMixed,
-        const bridge::RtpMap& rtpMap,
-        bool ssrcRewrite,
-        const uint32_t idleTimeoutSeconds,
+    EngineAudioStream(const std::string& endpointId_,
+        const size_t endpointIdHash_,
+        const uint32_t localSsrc_,
+        const utils::Optional<uint32_t>& remoteSsrc_,
+        transport::RtcTransport& transport_,
+        const bool audioMixed_,
+        const bridge::RtpMap& rtpMap_,
+        bool ssrcRewrite_,
```

Comment:
Did cpp check complain about this. Iirc the C++ standard resolves exactly these situations without postfixing etc.

Reply:
Yeah, unfortunately it did

Reply:
Can we suppress it because it is not a real error?